### PR TITLE
Create rql.md

### PR DESCRIPTION
**Pull request by hairmare**
Written as part of libgraviton/graviton#218...

### CODE REVIEW

**Comment by hairmare** [comment]
@narcoticfresh @lapistano let me know if you want me to add something...

**Comment by lapistano** [comment]
looks good for me

File: source/_pages/api/rql.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: File
```
Comment (narcoticfresh) [inline source/_pages/api/rql.md]: shouldn' this be 'rql'?

**Comment by narcoticfresh** [comment]
@hairmare what about the rather 'special' encoding of string values? shouldn't we also include this, how to encode `.,-` and so on inside string values? And what about casting values? That could be helpful too..

**Comment by eburi** [comment]
Found out that currently the limit-function in Graviton does not work as specified here: https://doc.apsstandard.org/2.1/spec/rql/#functions
Added a remark under Special-Cases

File: source/_pages/api/rql.md (hunk 1 above)
Comment (hairmare) [inline source/_pages/api/rql.md]: It should

**Comment by hairmare** [comment]
@narcoticfresh I added a link to upstreams encoding docs regarding the ``.,-`` issue.

**Pull request merged**